invenio_accounts/ext.py

```diff
@@ -248,21 +258,6 @@ def init_config(self, app):
             if any([k.startswith(prefix) for prefix in config_apps]):
                 app.config.setdefault(k, getattr(config, k))
 
-        # Set Session KV store
-        if app.config.get('ACCOUNTS_SESSION_REDIS_URL'):
```

Comment:
So since we want to keep the next version backward compatible, I would suggest to not remove the `ACCOUNTS_SESSION_REDIS_URL` and instead, if it is present then initialize it as before, but by giving priority to the `ACCOUNTS_SESSION_STORE_FACTORY`.